test/uk/gov/hmrc/merchandiseinbaggage/stubs/MibBackendStub.scala

```diff
@@ -132,7 +133,7 @@ object MibBackendStub extends MibConfiguration with CoreTestData {
   ): StubMapping =
     server
       .stubFor(
-        post(urlPathEqualTo(s"$calculationsUrl"))
+        post(urlPathEqualTo(s"${config.mibCalculationsUrl}"))
```

Comment:
This can just be `post(urlPathEqualTo("/declare-commercial-goods/calculations"))`